arb_os/precompiles.mini

```diff
@@ -256,6 +272,45 @@ public impure func precompile_0x05() {  // modexp
     evmOp_revert_knownPc(2, 0, 0);
 }
 
+func precompile_0x05_pluggable(
+    args: (EvmCalFrame, ByteArray, any),
```

Comment:
Looks like this is a typo? `EvmCalFrame`. Shouldn't this have failed compilation?

Reply:
Yeah, that's odd.  I raised the compiler issue with Robert, and fixed the typo.